examples/cf-worker-uploader-remover/serverless.yml

```diff
@@ -4,6 +4,6 @@ components:
   myWorker:
     type: cloudflare-workers
     inputs:
-      zoneId: 'c065856165e3fb015aa9bbc280663506'
+      zoneId: '66de06a610e659edd89a6de59df291d2'
```

Comment:
These should be example values 😄

Reply:
This can work as an example value, they can't push anything without his API key. Could change the nirvik.me to example.com though

Reply:
Agreed

Reply:
👍